scoap3/tasks.py

```diff
@@ -368,10 +370,19 @@ def _create_affiliation(data, authors):
                 "value": affiliation.get("value", ""),
                 "organization": affiliation.get("organization", ""),
             }
+            ror = affiliation.get("ror", "")
```

Comment:
lets strip the ROR, so to have only the id
so in `https://ror.org/00x194q47` we keep only `00x194q47` (remove `https://ror.org/`)